src/main/java/uk/gov/companieshouse/api/accounts/util/AccountsLogUtil.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.companieshouse.api.accounts.util;
+
+public enum AccountsLogUtil {
+    END_OF_REQUEST_MSG("End of Request"),
+    START_OF_RQUEST_MSG("Start Request"),
+    REQUEST_ID("X-Request-Id"),
+    ERIC_IDENTITY("Eric-Identity"),
+    START_TIME_KEY("START_TIME");
```

Comment:
The generic information (start time, request start and end messages) should be in a shared library (structured-logging) to be consistent across projects to help the support team.